### PR TITLE
updated comment at requests for shift_scheduling_sat.py

### PR DESCRIPTION
**Pull request by georgiossalon**

### CODE REVIEW

File: examples/python/shift_scheduling_sat.py [hunk 1]
```diff
@@ -215,11 +215,11 @@ def solve_shift_scheduling(params, output_proto):
     # Request: (employee, shift, day, weight)
     # A negative weight indicates that the employee desire this assignment.
     requests = [
-        # Employee 3 wants the first Saturday off.
+        # Employee 3 wants the first shift of Saturday off.
```
Comment (Mizux) [inline examples/python/shift_scheduling_sat.py:218]: I would rephrase it too to be consistency:
"Employee 3 does not want the first shift on the first Saturday (negative weight)."

Comment (lperron) [reply examples/python/shift_scheduling_sat.py:218]: and please against master

Comment (georgiossalon) [reply examples/python/shift_scheduling_sat.py]: @lperron The pull request goes into google:master. Does this fulfill your request?

Comment (Mizux) [reply examples/python/shift_scheduling_sat.py]: I guess laurent was trick by the name of your branch stable otherwise everything is good !
thanks for the PR

**Comment by lperron** [comment]
I will revert this. The direction is wrong. value > 0 means the shift is not desired.
value < 0 means the shift is desired.

**Comment by georgiossalon** [comment]
I am sorry about that. Then line 218 (Saturday off) should get corrected.

**Comment by Mizux** [comment]
fixed by https://github.com/google/or-tools/commit/6f24d2a0eaa7176cd0ff783bcff4d8c8139447f4
